tests/everest/test_everest_client.py

```diff
@@ -62,7 +63,17 @@ def ping_server() -> bool:
             else:
                 return True
 
-        wait_until(ping_server, timeout=timeout, interval=sleep_between_retries)
+        # These warnings emitted by uvicorn, which is still using legacy
+        # websockets. This is a know issue, and does not cause problems in the
```

Comment:
know -> known.

Is this a known issue in uvicorn? I could not find it in the issue list (but did not dig deep). If there is an issue, maybe include it in the comment.

Reply:
It is in their discussions: (see: https://github.com/encode/uvicorn/discussions/2476)

I have updated the PR to include this url.